homeassistant/components/trafikverket_weatherstation/coordinator.py

```diff
@@ -12,13 +13,14 @@
 )
 from pytrafikverket.trafikverket_weather import TrafikverketWeather, WeatherStationInfo
 
-from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_API_KEY
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
+if TYPE_CHECKING:
+    from . import TVWeatherConfigEntry
 from .const import CONF_STATION, DOMAIN
```

Comment:
```suggestion
from .const import CONF_STATION, DOMAIN

if TYPE_CHECKING:
    from . import TVWeatherConfigEntry
```
Just a suggestion: Maybe move the `if TYPE_CHECKING` block below all other imports. _I wish ruff would do that automatically._